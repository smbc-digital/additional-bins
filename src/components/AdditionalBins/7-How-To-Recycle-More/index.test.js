import React from 'react'
import { HowToRecycleMore } from './index'
import renderer from 'react-test-renderer'

describe('HowToRecycleMore',() => {
    describe('snapshot', () => {
        it('renders correctly', () => {
            const context = {}

            const tree = renderer
                .create(<HowToRecycleMore context={context} />)
                .toJSON()
                
            expect(tree).toMatchSnapshot()
        })
    })
})