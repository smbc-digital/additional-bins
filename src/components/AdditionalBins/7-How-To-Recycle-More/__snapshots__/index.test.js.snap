// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowToRecycleMore snapshot renders correctly 1`] = `
<form>
  <div
    className="ineligible-warning"
  >
    <div
      className="header-container"
    >
      <div>
        <i
          aria-hidden="true"
          className="fa fa-exclamation"
        />
      </div>
      <h1>
         We're sorry but you can't request an additional black bin
      </h1>
    </div>
    <h2>
      Why not?
    </h2>
    <p>
      You must recycle as much of your waste as possible.
    </p>
    <h3>
      What you can do next
    </h3>
    <p>
      To find out about what goes in each bin and how you can recycle as much waste as possible, visit our recycling pages using the button below.
    </p>
    <a
      className="button-primary"
      href="https://www.stockport.gov.uk/what-you-can-put-in-your-bins"
      id="what-you-can-put-in-your-bins"
    >
      What you can put in your bins
    </a>
  </div>
</form>
`;
